lumigator/python/mzai/backend/services/experiments.py

```diff
@@ -138,7 +138,7 @@ def create_experiment(self, request: ExperimentCreate) -> ExperimentResponse:
         # NOTE: Whenever the list of libraries below change, the LD_PRELOAD line in
         # `docker-compose-local.yaml` has to be updated with a new hash value
         runtime_env = {
-            "pip": ["nltk==3.8.1", "lm-buddy[jobs]==0.12.1"],
+            "pip": ["nltk==3.8.1", "datasets==2.20.0", "lm-buddy[jobs]==0.12.1"],
```

Comment:
@veekaybee a heads up that instead of updating `evaluate` to 0.4.3  in lm-buddy, I preferred to freeze `datasets` here so we won't touch lm-b's code until it is merged into lumigator